src/mailing/Mailing.Template/EmailTemplates/osp_welcome_email.html

```diff
@@ -0,0 +1,234 @@
+                  <tr>
+                    <td
+                      style="padding:0;Margin:0;padding-top:20px;padding-left:20px;padding-right:20px;text-align: left;">
+                      <table role="presentation" aria-hidden="true"
+                        style="mso-table-lspace:0pt;mso-table-rspace:0pt;border-collapse:collapse;border-spacing:0px;width:100%">
+                        <tr>
+                          <td style="padding:0;Margin:0;width:560px;text-align: left;">
+                            <table role="presentation" aria-hidden="true"
+                              style="mso-table-lspace:0pt;mso-table-rspace:0pt;border-collapse:collapse;border-spacing:0px;width:100%">
+                              <tr>
+                                <td
+                                  style="Margin:0;padding-top:15px;padding-bottom:15px;padding-left:20px;padding-right:20px;text-align: left;background-color: #ffffff;">
+                                  <p
+                                    style="Margin:0;-webkit-text-size-adjust:none;-ms-text-size-adjust:none;mso-line-height-rule:exactly;font-family:arial, 'helvetica neue', helvetica, sans-serif;line-height:29px;color:#333333;font-size:19px">
+                                    <strong>OSP Welcome email</strong>
+                                  </p>
+                                </td>
+                              </tr>
+                            </table>
+                          </td>
+                        </tr>
+                        <tr>
+                          <td style="padding:0;Margin:0;width:560px;text-align: center;">
+                            <table role="presentation" aria-hidden="true"
+                              style="mso-table-lspace:0pt;mso-table-rspace:0pt;border-collapse:collapse;border-spacing:0px;width:100%">
+                              <tr>
+                                <td
+                                  style="Margin:0;padding-top:20px;padding-bottom:20px;padding-left:30px;padding-right:30px;text-align: left;">
+                                  <p
+                                    style="Margin:0;-webkit-text-size-adjust:none;-ms-text-size-adjust:none;mso-line-height-rule:exactly;font-family:arial, 'helvetica neue', helvetica, sans-serif;line-height:21px;color:#333333;font-size:14px">
+                                    Dear {firstName} {lastName},<br /><br /> your registration at the Catena-X  dataspace got based on your request successfully generated by {osp}. <br /><br /> We have created your registration request. Before the registration validation is taking place, a final check from your side confirming your registration data and confirming the terms & conditions is needed. <br /><br /> Please follow the link below to access your registration data and to confirm the company role related terms & conditions. <br /> You may want to update the company roles by selecting additional roles.</p>
```

Comment:
could we please change the part, than we could set that in the backend and if the first or lastname is null we could set a generic value.

```suggestion
                                    Dear {username},<br /><br /> your registration at the Catena-X  dataspace got based on your request successfully generated by {osp}. <br /><br /> We have created your registration request. Before the registration validation is taking place, a final check from your side confirming your registration data and confirming the terms & conditions is needed. <br /><br /> Please follow the link below to access your registration data and to confirm the company role related terms & conditions. <br /> You may want to update the company roles by selecting additional roles.</p>
```